sonar-ws/src/main/java/org/sonarqube/ws/client/setting/SetRequest.java

```diff
@@ -42,11 +45,17 @@ public String getKey() {
     return key;
   }
 
+  @CheckForNull
   public String getValue() {
     return value;
   }
 
   @CheckForNull
```

Comment:
I think not, except if we want to return an empty list and checks it fails if we add an empty list